homepage/homepage/start.sh

```diff
@@ -12,7 +12,9 @@ if ! grep github.com "${SSH_KNOWNHOSTS_FILE}" > /dev/null
 then
      echo "github.com ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQCj7ndNxQowgcQnjshcLrqPEiiphnt+VTTvDP6mHBL9j1aNUkY4Ue1gvwnGLVlOhGeYrnZaMgRK6+PKCUXaDbC7qtbW8gIkhL7aGCsOr/C56SJMy/BCZfxd1nWzAOxSDPgVsmerOBYfNqltV9/hWCqBywINIR+5dIg6JTJ72pcEpEjcYgXkE2YEFXV1JHnsKgbLWNlhScqb2UmyRkQyytRLtL+38TGxkxCflmO+5Z8CSSNY7GidjMIZ7Q4zMjA2n1nGrlTDkzwDCsw+wqFPGQA179cnfGWOWRVruj16z6XyvxvjJwbz0wQZ75XK5tKSb7FNyeIEs4TT4jk+S4dhPeAUC5y+bDYirYgM4GC7uEnztnZyaVWQ7B381AK4Qdrwt51ZqExKbQpTUNn+EjqoTwvqNj4kqx5QUCI0ThS/YkOxJCXmPUWZbhjpCg56i+2aB6CmK2JGhn57K5mj0MNdBXA4/WnwH6XoPWJzK5Nyu2zB3nAZp+S5hpQs+p1vN1/wsjk=" >> "${SSH_KNOWNHOSTS_FILE}"
 fi
-## Pull domain from HOMEPAGE_TEMPLATE_REPO.
+## Pull domain from HOMEPAGE_TEMPLATE_REPO, which can be in 1 of 2 formats:
+## it@github.com:YourUsername/my-private-homepage-template.git
```

Comment:
missing g

Reply:
I'll fix the typo but does it make sense? Is there a better way?

Reply:
I guess you should explain why, https=public git@=private and requires deploy key.

Reply:
Oh! If the deploy key is only requred for HOMEPAGE_TEMPLATE_REPO=git@, then I can simplify this by: `if HOMEPAGE_TEMPLATE_REPO=git@, grep domain:port in known_hosts and if it doesn't exist then ssh_keyscan [domain]:port >> known_hosts`.

I can completely disregard if HOMEPAGE_TEMPLATE_REPO=https

Does that sound right?

Reply:
Sounds right. Now a user could have an https repo that still requires authentication (ie. Not public repo) but we won't support that. For our purposes https is public and requires no deploy key.

Reply:
fixed typo, changed script to only add HOMEPAGE_TEMPLATE_REPO to known_hosts if it needs a deploy key (i.e., HOMEPAGE_TEMPLATE_REPO is in the format `git@github.com:YourUsername/my-private-homepage-template.git`)